<launch>
  <!-- Fusion -->
  <node pkg="fusion" output="log" type="merge.py" name="topic_merger" respawn="False">
    <param name="base_link_frame" value="ugr/car_base_link" />
    <remap from="/input/lidar_observations" to="/ugr/car/observations/lidar"/>
    <remap from="/input/camera_observations" to="/ugr/car/observations/camera"/>
    <remap from="/output/topic" to="/ugr/car/observations/fused"/>
  </node>
</launch>